CHANGES.md

```diff
@@ -59,6 +59,7 @@ Version PBFT
  -  Added `BlockMarshaler.UnmarshalBlockHash()` method. [[#PBFT]]
  -  Added `BlockChain<T>.GetBlockCommit()` method.  [[#PBFT]]
  -  Added `BlockChain<T>.CleanupBlockCommitStore()` method.  [[#PBFT]]
+ -  Added `BlockChain<T>.ValidateBlockCommit()` method  [[#PBFT]]
```

Comment:
Is there a need for this to be `public`?

Reply:
Amended. I've changed it to `internal` for testing.